### ISSUE TITLE
Can't create philosophers stones

### ISSUE THREAD

**Issue by weschaos** [Gstayton/GeometricMagic#1]
Cause: I think fire doesn't exist anymore. Whenever I kill a bunch of villagers with the circle, they all drop smooth stone.

No errors in the console relevant to geometric magic. I can't give myself fire blocks, either.

version 2.8.6 and spigot 1.8

Plugin may require an additional, unobtainable block to replace fire. Not sure how to facilitate that, however.

**Comment by Gstayton** [comment]
Glad to hear someone is actively using this mod; I was in the middle of a total re-write when the whole DMCA against bukkit happened. But the issue that you're talking about seems small enough, I'd be willing to push an update to change the item used for philo stones, if you've got an alternative item to use, as I haven't kept up too closely with the new items.

**Comment by weschaos** [comment]
Well, I did a little reading and apparently technical blocks in general don't exist anymore.

I don't know much about coding plugins, but what could be done is assign the philospher's stone to drop from one of the unused potion ID's if such a thing is possible. Hopefully, if other servers decide to install your plugin that it won't conflict with what they already have.

Also, in addition, I would suggest a reevaluation of the value of blocks. I don't know if it's because I had OP that it gave me so much xp, but turning a diamond block into air gives you somewhere in the neighborhood of 73k levels, iron gives about 3k, and basic cobble gives one. Most of my peers agree with me that there's no way to balance that kind of level gain with basic enchanting. I know that I can change that manually in the configuration files, but I'm uncertain if I can make cobble and other similarly common materials give fractions of levels. Simply, it's too easy to just grab a bunch of cobblestone, convert it to air and then grind enchantments.

Thank you for your reply, and I hope that these issues can be resolved.

edit: I feel like I'm imposing asking so much. Would it be possible to give up alchemy or something in addition to crafting in order to gain a bunch of permanent potion effects, and say they became a Homunculus? I always really like the Homunculi.

**Comment by Gstayton** [comment]
Believe me, I had the intention of revaluing all the blocks, but again, I didn't expect anyone to still be running any bukkit servers.

As for the value of blocks, no there is no way to make it work like you want. There is compatibility for an economy plugin, however, to avoid using levels. The one thing I had finished working on in the revamp of the mod was integrating it's own value tracking, instead of relying on economy plugins/XP.

Really, I wasn't planning on putting more work into this until SpongeCraft was released. But if someone is actively using it, I suppose I can dive back into the codebase, but it really is a mess. The value system is one of the easier parts of the original code to work with, however.

Edit: However, I did just realize that I can't get a copy of any of the reference jars for the latest version, and I am obviously at least a version out of date. Which... Puts a damper on things.
